packages/react-components/react-motion-components-preview/library/src/types.ts

```diff
@@ -50,6 +50,17 @@ export type PresenceEasing = {
   exitEasing?: string;
 };
 
+/**
+ * Common delay parameters for presence motion components.
+ */
+export type PresenceDelay = {
```

Comment:
Every single presence component share this and other Presence related partial prop types, I'd suggest doing the same thing you did with the base atom types to presence